libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/TriggerConditions/Microsoft.OnEndOfConversationActivity.schema

```diff
@@ -6,9 +6,9 @@
     "type": "object",
     "required": [
     ],
-    "policies": [
+    "$policies": [
```

Comment:
>"$policies":  [](start = 4, length = 13)

Should be $policies: { notInteractive: true}

Reply:
#fixed